src/Custom-sample.h

```diff
@@ -424,49 +424,49 @@ static const char DATA_ESPEASY_DEFAULT_MIN_CSS[] PROGMEM = {
 // #define USES_P070   // NeoPixel_Clock
 // #define USES_P071   // Kamstrup401
 // #define USES_P072   // HDC1000/HDC1008/HDC1010/HDC1050/HDC1080
-// #define USES_P073   // 7DG
+// #define USES_P073   // 7-segment display
 // #define USES_P074   // TSL2591
 // #define USES_P075   // Nextion
-// #define USES_P076   // HWL8012   in POW r1
-// #define USES_P077   // CSE7766   in POW R2
-// #define USES_P078   // Eastron Modbus Energy meters
-// #define USES_P079   // Wemos Motoshield
+// #define USES_P076   // HLW8012/BL0937
+// #define USES_P077   // CSE7766
```

Comment:
I think the "in POW Rxx" is useful here.

Reply:
All the "description" are taken from the official Device page https://espeasy.readthedocs.io/en/latest/Plugin/_Plugin.html

Reply:
This one should only be in the Custom.h sample as it is just a hint on which one to enable/disable when making a custom build.